.github/ISSUE_TEMPLATE/web_api_type_definition_issue.yml

```
@@ -51,6 +51,15 @@ body:
         - label: "I am using the latest version of TS"
     validations:
       required: true
+  - type: checkbox
+    id: use_latest_typescript_web
+    attributes:
+      label: "Use the latest @types/web Package"
+      description: "Check this if you are using the latest @types/web package."
```

We don't have to require usage, just trying it before reporting would be fine. Same for the TS version above.

```suggestion
description: "Check this if you have tried the latest `@types/web` package."
```